app/templates/power-ups/client-library.html

```diff
@@ -86,45 +50,116 @@
+  <li><strong>localizeKeys(keys)</strong> Localize a list of strings or objects. You can read more about internationalizationa and localization in the <a ui-sref="power-ups.topics({'#':'internationalization'})">topics section</a>.</li>
+  <li><strong>board(field1, field2, ...)</strong>
+	Returns a promise with information about the board for the current context.
 	Valid fields include: 'id', 'name', 'url', 'shortLink'</li>
-	<li><strong>list([requestedField1],[requestedField2],[…])</strong> Returns a promise with information about the list for the current context
+	<li><strong>list(field1, field2, ...)</strong>
+	Returns a promise with information about the list for the current context
 	Valid fields include: 'id', 'name'</li>
-	<li><strong>card([requestedField1],[requestedField2],[…])</strong> Returns a promise with information about the card for the current context
-	Valid fields include: 'id', 'name', 'desc', 'due', 'cover', 'attachments', 'members', 'labels', 'url', 'shortLink'</li>
-	</ul>
-	<p>
-	<li><strong>t.closePopup</strong>, <strong>t.closeOverlay</strong>, <strong>t.iframe</strong> can only be called within an appropriate context.
-	</p>
-	<p>
+	<li><strong>card(field1, field2, ...)</strong>
+	Returns a promise with information about the card for the current context. The information available is only when a specific card is in context. For instance, this function will return information about a card successfully inside of the capabilities 'attachment-sections' and 'card-badges', but not 'show-settings' nor 'board-buttons'. The latter are not within the context of a single card.
+	Valid fields include: 'id', 'name', 'desc', 'due', 'closed', 'cover', 'attachments', 'members', 'labels', 'url', 'shortLink', 'idList'<br /><br />
+	Example usage:
+<pre>t.card('id', 'name', 'url').then(function(promiseResult){console.log(promiseResult)});
+// <- {id:"72hejh3iuruyr87rhiu", name: "Card Name", url: "https://trello.com/c/38UCRu1a/card-name"}</pre>
+	</li>
+	<li><strong>t.closePopup</strong>, <strong>t.closeOverlay</strong>, <strong>t.iframe</strong> can only be called within an appropriate context.</li>
 	<li><strong>t.popup</strong> can only be called within a capability, not currently from an iframe such as an attachment-section. This is due to the relative placement of the popup to an item within the Trello Web Client.</li>
-	</p>
-
+</ul>
 
+<h3 id="capability-options">Capability Options</h3>
+<p>Each capability that you include in your initialization will take a reference to the <strong>t</strong> object and a options or context object. This object will include various things depending on the capability.</p>
+  <p>Board Capability Options</p>
+  <pre>{
+  "context":{
+    "board":"55db14fd3e105ac84105bd75",
+    "command":"board-buttons",
+    "plugin":"564ddf493f183488ea5ddc0e"
+  },
+  "locale":"en-US"
+}</pre>
+  <p>Card Capability Options</p>
+  <pre>{
+  "context":{
```

Comment:
This has changed a bit and looks like:

```json
"{
  "context": {
    "member": "556c8537a1928ba745504ad8",
    "permissions": {
      "board": "write",
      "organization": "write",
      "card": "write"
    },
    "board": "5834bb5fb7d8021c5b51a8de",
    "card": "5858482d9c1c35c230d475ba",
    "command": "card-buttons",
    "__insecure_signature": "69eda4985e3fa8c7286cf626-f65bc7b"
  },
  "locale": "en-US"
}"
```

(The changes here are the addition of the member, permissions, and insecure signature, which are there in all contexts